modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
-| Read Capacity Units a| Number |  The maximum number of strongly consistent reads consumed per second before DynamoDB returns a ThrottlingException. |  |x
-| Write Capacity Units a| Number |  The maximum number of writes consumed per second before DynamoDB returns a ThrottlingException. |  |x
-| Global Secondary Indexes a| Array of GlobalSecondaryIndex |  One or more global secondary indexes (the maximum is five) to be created on the table. |  |
-| Local Secondary Indexes a| Array of LocalSecondaryIndex |  One or more local secondary indexes (the maximum is five) to be created on the table. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Table Name a| String |  The name of the table to create. |  | x
+| Attribute Definitions a| Array of <<AttributeDefinition>> |  An array of attributes that describe the key schema for the table and indexes. |  | x
+| Key Schemas a| Array of <<KeySchemaElement>> |  Specifies the attributes that make up the primary key for a table or an index. |  | x
+| Read Capacity Units a| Number |  The maximum number of strongly consistent reads consumed per second before DynamoDB returns a ThrottlingException. |  | x
+| Write Capacity Units a| Number |  The maximum number of writes consumed per second before DynamoDB returns a ThrottlingException. |  | x
+| Global Secondary Indexes a| Array of <<GlobalSecondaryIndex>> |  One or more global secondary indexes (the maximum is five) to be created on the table. |  | 
+| Local Secondary Indexes a| Array of <<LocalSecondaryIndex>> |  One or more local secondary indexes (the maximum is five) to be created on the table. |  | 
 | Stream View Type a| Enumeration, one of:
 
-** KEYS_ONLY
-** NEW_AND_OLD_IMAGES
 ** NEW_IMAGE
 ** OLD_IMAGE
-|  When an item in the table is modified, StreamViewType determines what information is written to the table's stream. |  |
-| Stream Enabled a| Boolean |  Indicates whether Streams is to be enabled (true) or disabled (false). |  false |
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+** NEW_AND_OLD_IMAGES
+** KEYS_ONLY |  When an item in the table is modified, StreamViewType determines what information is written to the table's stream. |  | 
+| Stream Enabled a| Boolean |  Indicates whether Streams is to be enabled (true) or disabled (false). |  false | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
An expression to evaluate against the operation's output. The evaluation outcome is stored in the target variable.